packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -23,28 +23,30 @@ import {
  */
 export interface ITree extends IChannel {
 	/**
-	 * Returns a tree that is known to be compatible with the provided schema. The returned tree exposes an API that is schema-aware.
+	 * Returns a {@link TreeView} using the provided schema.
+	 * If the tree's stored schema is compatible, this will provide a schema aware API for accessing the tree's content.
+	 * If the provided schema is incompatible with the stored data, the view will instead expose an error indicating the incompatibility.
```

Comment:
Are you imagining we'll want to expand the error type you've presented in the API in this PR? This comment makes me think yes and I think that decision is OK, just want to double-check I'm on the same page here.